src/goto-programs/interpreter.cpp

```diff
@@ -679,9 +681,9 @@ void interpretert::execute_assign()
     side_effect_exprt side_effect=to_side_effect_expr(code_assign.rhs());
     if(side_effect.get_statement()==ID_nondet)
     {
-      unsigned address=integer2unsigned(evaluate_address(code_assign.lhs()));
-      size_t size=get_size(code_assign.lhs().type());
-      for(size_t i=0; i<size; i++)
+      std::size_t address=integer2unsigned(evaluate_address(code_assign.lhs()));
```

Comment:
should be `integer2size_t`

Reply:
somehow related to this PR, there are also occurrences of `size_t` and not `std::size_t`